src/api/urls.py

```diff
@@ -20,118 +20,106 @@
-                                        url(
-                                            r"^(?:extras|offers)/?$",
-                                            offer_list_feedback.offer_list_feedback,
-                                            name="api_offer_list_feedback",
-                                        ),
-                                        url(
-                                            r"^(?:extra|offer)/?$",
-                                            offer_feedback.offer_feedback,
-                                            name="api_offer_feedback",
-                                        ),
-                                    ]
-                                ),
-                            ),
-                        ]
-                    ),
+                    r"^$",
+                    legacy_feedback_endpoint.legacy_feedback_endpoint,
+                    name="api_legacy_feedback_endpoint",
+                ),
+                url(
+                    r"^categories/?$",
+                    region_feedback.region_feedback,
+                    name="api_region_feedback",
+                ),
+                url(
+                    r"^page/?$",
+                    page_feedback.page_feedback,
+                    name="api_page_feedback",
+                ),
+                url(
+                    r"^poi/?$",
+                    poi_feedback.poi_feedback,
+                    name="api_poi_feedback",
+                ),
+                url(
+                    r"^event/?$",
+                    event_feedback.event_feedback,
+                    name="api_event_feedback",
+                ),
+                url(
+                    r"^events/?$",
+                    event_list_feedback.event_list_feedback,
+                    name="api_event_list_feedback",
+                ),
+                url(
+                    r"^imprint-page/?$",
+                    imprint_page_feedback.imprint_page_feedback,
+                    name="api_imprint_page_feedbacks",
+                ),
+                url(
+                    r"^map/?$",
+                    map_feedback.map_feedback,
+                    name="api_map_feedback",
+                ),
+                url(
+                    r"^search/?$",
+                    search_result_feedback.search_result_feedback,
+                    name="api_search_result_feedback",
+                ),
+                url(
+                    r"^(?:extras|offers)/?$",
+                    offer_list_feedback.offer_list_feedback,
+                    name="api_offer_list_feedback",
+                ),
+                url(
+                    r"^(?:extra|offer)/?$",
+                    offer_feedback.offer_feedback,
+                    name="api_offer_feedback",
                 ),
             ]
         ),
     ),
 ]
 
+legacy_wp_urlpatterns = [
+    url(r"^children/?$", children, name="api_childrens"),
+]
+
+
```

Comment:
```suggestion
```
I don't get why the children endpoint is declared as "legacy wp endpoint" and only made available in the old url schema... If the children endpoint is actually used, it will also be used in the future and should probably also be available in the "new" Django api naming scheme (as part of the `content_api_urlpatterns`), right?